amethyst_renderer/src/pipe/effect/pso.rs

```diff
@@ -79,6 +82,18 @@ impl<'d> PipelineInit for Init<'d> {
             meta.out_colors.push(meta_color);
         }
 
+        for blend in self.out_blends.iter() {
+            let mut meta_blend = <BlendTarget as DataLink<'d>>::new();
+            for info in info.outputs.iter() {
+                if let Some(res) = meta_blend.link_output(info, blend) {
+                    let d = res.map_err(|e| InitError::PixelExport(info.name.as_str(), Some(e)))?;
+                    desc.color_targets[info.slot as usize] = Some(d);
+                    break;
```

Comment:
You could just use `find` instead of a for loop.

Reply:
This ends up being less efficient, as you then have to call `link_output` twice.